corehq/apps/app_manager/templates/app_manager/partials/add_item_templates.html

```diff
@@ -0,0 +1,126 @@
+{% load i18n %}
+
+<script type="text/html" id="js-popover-template-add-item">
+  <div class="popover popover-additem" role="tooltip"><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>
+</script>
+
+<script type="text/html" class="js-popover-template-add-item-content" data-slug="module">
```

Comment:
nit
```
<script
  type="text/html"
  class="js-popover-template-add-item-content"
  data-slug="module"
>
```

because of the data attribute

Reply:
There's a prettify PR open to reformat these files: https://github.com/dimagi/commcare-hq/pull/35630

Reply:
perfect! then ignore all my "nits"